Automattic/woocommerce-payments
# Refactor `AccountBalances` card to allow separate welcome / balances cards

Fixes #6360

#### Changes proposed in this Pull Request









This PR allows the `AccountBalances` card to be split into 2 cards.

1. The `AccountBalancesHeader` component has been renamed to `Welcome`.
2. `Welcome` code is now located in a distinct component directory: `client/components/welcome/` since it is now decoupled from the `AccountBalances` card.
3. The `AccountBalances` component is now simpler: it no longer includes the `<Card>` wrap or logic related to the `Welcome` component. This is now handled in `overview/index.js`.
4. Unit tests for `AccountBalances` and `Welcome` have been improved or updated.

> **Note**
> Note for reviewer: we could achieve the outcome without change 1 and 2 (renaming & moving the Header component to a distinct component), but I think it makes sense to separate the Header (greeting) component from the Balances component in file structure.

**Before**

![image](https://github.com/Automattic/woocommerce-payments/assets/3147296/9912a01b-4f23-46b7-be53-50e379326836)

**After**

![image](https://github.com/Automattic/woocommerce-payments/assets/3147296/ce01b2eb-e0d6-440e-983e-5c8cb74a6d3e)

#### Testing instructions






* Visit Payments → Overview
* If your store has active disputes or not, you should see the existing combined Welcome/Balances card (see screenshot **After**)
* Ensure all tests pass
* Manually edit the Overview screen code ([code ref](https://github.com/Automattic/woocommerce-payments/blob/8c4c85fb08b1f26a3e206fde65b815abf6c15833/client/overview/index.js#L146-L153)) to test that the components work as separate cards:

```tsx
<>
	<Card>
		<Welcome />
	</Card>
	<Card>
		<AccountBalances />
	</Card>

	<DepositsOverview />
</>
```

-------------------


**Post merge**
